From: Modestas Vainius <modax@debian.org>
Subject: Do not set install RPATH to /usr/lib by default
Forwarded: no
Origin: vendor
Last-Update: 2011-02-20

Use proper code in order to avoid setting INSTALL_RPATH to /usr/lib. It is
disallowed on Debian systems and is a bad practise in general.

--- a/cmake/FindPhononInternal.cmake
+++ b/cmake/FindPhononInternal.cmake
@@ -233,7 +233,10 @@ endif(APPLE)
 # RPATH directories outside the current CMAKE_BINARY_DIR and also the library 
 # install directory. Alex
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH  TRUE)
-set(CMAKE_INSTALL_RPATH "${LIB_INSTALL_DIR}")
+list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${LIB_INSTALL_DIR}" _isSystemLibDir)
+if("${_isSystemLibDir}" STREQUAL "-1")
+   set(CMAKE_INSTALL_RPATH "${LIB_INSTALL_DIR}")
+endif("${_isSystemLibDir}" STREQUAL "-1")
 
 
 # Uninstall Target
